src/routes/index.js

```diff
@@ -1,9 +1,14 @@
 import { Router } from 'express';
 import api from './api';
+import authRoutes from './api/auth';
+import profileRoutes from './api/profile';
 
 
 const router = new Router();
 
+router.use('/api/v1/auth', authRoutes);
+router.use('/api/v1/profiles', profileRoutes);
```

Comment:
Hi @darasimiolaifa, This is not following the existing routing pattern in the develop branch, please ensure you rebase develop